packages/core/strapi/bin/strapi.js

```diff
@@ -95,12 +95,13 @@ program
   .option('--dbssl <dbssl>', 'Database SSL')
   .option('--dbfile <dbfile>', 'Database file path for sqlite')
   .option('--dbforce', 'Allow overwriting existing database content')
+  .option('-ts, --typescript', 'Create a typescript project')
   .description('Create a new application')
   .action(require('../lib/commands/new'));
 
 // `$ strapi start`
 program
-  .command('start')
+  .command('start [dir]')
```

Comment:
probably better to set the cwd when running the command instead of supporting this 🤔  that avoids adding params for user that we wouldn't necessarily want to add straight away